### PR TITLE
iputils: update to 20190515.

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

File: srcpkgs/iputils/template [hunk 1]
```diff
@@ -1,39 +1,28 @@
...
-
-do_install() {
-	vbin ping ${pkgname}-ping
-	for f in clockdiff arping rdisc tracepath; do
-		vbin ${f}
-	done
-	for f in clockdiff arping ping rdisc tracepath; do
-		vman doc/${f}.8
-	done
+post_install() {
```
Comment (xtraeme) [inline srcpkgs/iputils/template]: `iputils-ping.8` exists but `iputils-ping` for alternatives does not.

Comment (CameronNemo) [reply srcpkgs/iputils/template]: fixed

File: srcpkgs/iputils/template [hunk 2]
```diff
@@ -1,39 +1,29 @@
...
-distfiles="https://github.com/${pkgname}/${pkgname}/archive/s${version}.tar.gz"
-checksum=da14105291dd491f28ea91ade854ed10aee8ba019641c80eed233de3908be7c5
+distfiles="https://github.com/iputils/iputils/archive/s${version}.tar.gz"
+checksum=9b5125eb5ef9f4e947ad8fdddcf77f538f53b8f47b53eb5bc5347cb16d01c8fd
 
 alternatives="
  ping:ping:/usr/bin/${pkgname}-ping
  ping:ping.8:/usr/share/man/man8/${pkgname}-ping.8
  ping6:ping6:/usr/bin/${pkgname}-ping
  ping6:ping6.8:/usr/share/man/man8/${pkgname}-ping.8"
```
Comment (xtraeme) [inline srcpkgs/iputils/template]: ping6 alternative still exists... I think it's redundant because ping(8) has -4 and -6 options.

Comment (CameronNemo) [reply srcpkgs/iputils/template]: Not sure what the problem is here. ping6 is pointing to iputils-ping... do we no longer wish to provide a ping6 compat link?

Comment (xtraeme) [reply srcpkgs/iputils/template]: IMHO this ping6 symlink is just redundant but I don't care enough to be opposed to keep it.

**Pull request merged**